src/api/spec/models/notification_spec.rb

```diff
@@ -10,4 +10,18 @@
     it { expect(subject.class).to eq(delete_package_event.class) }
     it { expect(subject.payload).to eq(delete_package_event.payload) }
   end
+
+  describe 'relationship with users' do
```

Comment:
:thought_balloon:  I don't see much sense in those kinds of tests, there is no logic involved and I'm pretty sure that the `ActiveRecord` test suite covers associations :-)